--- my_custom.dts	2022-04-18 17:01:13.183615927 +0000
+++ my_custom_new.dts	2022-04-18 17:02:15.128205045 +0000
@@ -99,3 +99,22 @@
 &usb1 {
 	status = "okay";
 };
+&fpga_bridge0 {
+  status = "okay";
+  bridge-enable = <1>;
+};
+
+&fpga_bridge1 {
+  status = "okay";
+  bridge-enable = <1>;
+};
+
+&fpga_bridge2 {
+  status = "okay";
+  bridge-enable = <1>;
+};
+
+&fpga_bridge3 {
+  status = "okay";
+  bridge-enable = <1>;
+};
